### PR TITLE
Issue 25 fixed: Save changes from calendar to availability table in database

### PR DESCRIPTION
**Pull request by katja-maucec**
Deletes old availability and adds new availability in 1 hour increments based on shifts highlighted in dynamic calendar.

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -4,9 +4,6 @@ var bodyParser = require("body-parser");
 
 var exampleRoutes = require("./routes/example");
 var calendarRoutes = require("./routes/calendar");
-var employeesRoutes = require("./routes/employees");
```
Comment (bdeleonardis1) [inline server/app.js]: This looks wrong to me. I think you were trying to resolve some merge conflicts but we don't want to delete these routes! You can just copy and paste the code github is saying was deleted back into the file.

File: server/app.js [hunk 2]
```diff
@@ -16,9 +13,6 @@ app.use(cors());
 
 app.use("/example", exampleRoutes);
 app.use("/", calendarRoutes);
-app.use("/", employeesRoutes);
```
Comment (bdeleonardis1) [inline server/app.js]: Same here.

File: server/routes/calendar.js [hunk 3]
```diff
@@ -1,6 +1,42 @@
 var express = require("express");
 var router = express.Router();
 
+var savedAvails = [];
+router.post("/save", (req, res) => {
```
Comment (bdeleonardis1) [inline server/routes/calendar.js]: Is this endpoint actually doing anything? Can we delete? It looks like the endpoint below this is what is actually doing the saving to the database.

File: server/routes/calendar.js [hunk 4]
```diff
@@ -1,6 +1,42 @@
...
+  items = req.body.items;
+  console.log(items);
+  savedAvails = items;
+  return res.json({ schedule: items });
+});
+
+router.get("/availability", function(req, res) {
+  pool.query("SELECT * FROM AVAILABILITY", (error, result) => {
```
Comment (bdeleonardis1) [inline server/routes/calendar.js]: This is unnecessary right? We're not doing anything with the information from the availability table are we?

**Comment by kerryhuang1** [comment]
Ready for another look

**Pull request merged**